sink.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * @module  web-audio-stream/sink
+ *
+ * Sink pull-stream for web-audio
+ */
+'use strict';
+
+
+var pull = require('pull-stream/pull');
+var asyncMap = require('pull-stream/throughs/async-map');
+var drain = require('pull-stream/sinks/drain');
+var createWriter = require('./writer');
+
+module.exports = sink;
+
+
+function sink (node, options) {
+	let write = createWriter(node, options);
+
+	let stream = pull(
+		asyncMap(write),
+		drain()
+	);
+
+	stream.abort = write.end;
```

Comment:
should this call the drain stream's `.abort`, or does this work as is? i have a feeling it wouldn't end the stream till the next time it's trying to write.